src/toil_scripts/adam_uberscript/adam_uberscript.py

```diff
@@ -124,53 +129,58 @@ def launch_pipeline(params):
             assert False, 'Invalid ref genome %s' % params.reference_genome
 
         # Assemble pipeline command to be stuffed into a screen session
-
+        
         pipeline_command = ['PYTHONPATH=$PYTHONPATH:~/toil-scripts/src',
                             'python -m toil_scripts.adam_gatk_pipeline.align_and_call',
                             'aws:{region}:{j}',
                             '--autoscale_cluster',
-                            '--sequence_dir {sequence_dir}',
-                            '--dir_suffix /{genome}',
                             '--retryCount 1',
-                            '--s3_bucket {b}',
-                            '--bucket_region {region}',
-                            '--uuid_manifest ~/manifest',
-                            '--ref {ref}',
-                            '--amb {amb}',
-                            '--ann {ann}',
-                            '--bwt {bwt}',
-                            '--pac {pac}',
-                            '--sa {sa}',
-                            '--fai {fai}',
                             '--use_bwakit',
-                            '--num_nodes {s}',
                             '--driver_memory {m}',
                             '--executor_memory {m}',
-                            '--phase {phase}',
-                            '--mills {mills}',
-                            '--dbsnp {dbsnp}',
-                            '--omni {omni}',
-                            '--hapmap {hapmap}',
                             '--batchSystem mesos',
                             '--mesosMaster $(hostname -i):5050',
                             '--workDir /var/lib/toil',
-                            '--file_size {fs}',
                             '--logInfo']
+        if mock_mode():
+            pipeline_command = ['ADAM_GATK_MOCK_MODE=1'] + \
+                               pipeline_command + \
+                               ['--dir_suffix /mock']
+        else:    
+            pipeline_command += ['--s3_bucket {b}',
```

Comment:
Those are passed in automatically by the pipeline from a dictionary. I did it that way so that running pipeline tests in mock mode without the überscript would be really straightforward.